indico/htdocs/sass/modules/abstracts/_abstract_page.scss

```diff
@@ -218,9 +231,24 @@
 }
 
 .abstract-comment-badges {
-    align-self: flex-end;
+    margin-right: auto;
```

Comment:
I think it's better to reason in terms of which element consumes horizontal space without mixing it with margins. 

Using `flex-grow` allows for adding, for instance, `margin-right: 1em` to make sure that no matter how much the content grows there will always be some space in between.